### ISSUE TITLE
how to add attendee/guests to calendar

### ISSUE THREAD

**Issue by nikiiTrivedi** [josephj/react-icalendar-link#6]
I tried to check [iCalendar docuemntation](https://icalendar.org/CalDAV-Scheduling-RFC-6638/b-1-example-organizer-inviting-multiple-attendees.html) and then try to create an event like this

```
 const event = {
      title: "My Title",
      description: "My Description",
      startTime: "2021-04-20T10:30:00+05:30",
      endTime: "2021-04-20T12:00:00+05:30",
      location: "Jamangar",
      guests/attendees: "person@mail.com"
    }
```

**Comment by josephj** [comment]
@nikiiTrivedi Sorry I haven't provided a way to add attendees. I am thinking to provide the following ways

For the string like this. 

```
ATTENDEE;CN="Cyrus Daboo";CUTYPE=INDIVIDUAL;PARTSTAT=ACCEPTED:
 mailto:cyrus@example.com
ATTENDEE;CN="Wilfredo Sanchez Vega";CUTYPE=INDIVIDUAL;PARTSTAT
 =NEEDS-ACTION;ROLE=REQ-PARTICIPANT;RSVP=TRUE:mailto:wilfredo@
 example.com
ATTENDEE;CN="Bernard Desruisseaux";CUTYPE=INDIVIDUAL;PARTSTAT=
 NEEDS-ACTION;ROLE=REQ-PARTICIPANT;RSVP=TRUE:mailto:bernard@ex
 ample.net
ATTENDEE;CN="Mike Douglass";CUTYPE=INDIVIDUAL;PARTSTAT=NEEDS-A
 CTION;RSVP=TRUE:mailto:mike@example.org
```
**Easy Version**
All their status wiell be `;CUTYPE=INDIVIDUAL;PARTSTAT=NEEDS-ACTION;ROLE=REQ-PARTICIPANT;RSVP=TRUE:`

```js
attendees: [
  "Cyrus Daboo<cyrus@example.com>"
  ...
]
```

**Detailed**

```js
attendees: [
  {
    name: "Cyrus Daboo",
    userType: "Individual",
    participationStatus: "needAction",
    role: "participant",
    rsvp: true, 
    email: "cyrus@example.com"
  }
]
```

**Comment by josephj** [comment]
@nikiiTrivedi I just published the version `2.0.1`. You can use the `rawContent` to achieve this. 

https://github.com/josephj/react-icalendar-link#using-raw-content

**Comment by josephj** [comment]
@nikiiTrivedi 

Thanks for @tingchun0113 's help. Now you can use the `attendees` field. 

```js
import React from "react";
import ICalendarLink from "react-icalendar-link";

class App extends React.Component {
  render() {
    const event = {
      title: "My Title",
      description: "My Description",
      startTime: "2018-10-07T10:30:00+10:00",
      endTime: "2018-10-07T12:00:00+10:00",
      location: "10 Carlotta St, Artarmon NSW 2064, Australia",
      attendees: [
        "Hello World <hello@world.com>",
        "Hey <hey@test.com>",
      ]
    }
    return (
      <ICalendarLink event={event}>
        Add to Calendar
      </ICalendarLink>;
    );
  }
}
```
